backend/app2/llm/agents/music_agent.py

```
@@ -90,6 +89,8 @@
 from app2.llm.streaming import TextDeltaEvent # Import TextDeltaEvent
 
 from app2.core.config import settings
+# Import request_manager to link ChatSession for cancellation
+from app2.sse.request_manager import request_manager 
```

Ruff (0.8.2)

`app2.sse.request_manager.request_manager` imported but unused

Remove unused import: `app2.sse.request_manager.request_manager`

(F401)